R/proj.R

```diff
@@ -214,8 +214,10 @@ check_is_package <- function(whos_asking = NULL) {
 
   message <- "Project {.val {project_name()}} is not an R package."
   if (!is.null(whos_asking)) {
+    # remove brackets, so that it is recognized as a help link
```

Comment:
You can remove the comment (or you need to say "parentheses", but better to just remove it).